WIP WIP WIP WIP

stevendanna opened the pull request.

stevendanna committed:
kvserver: move some benchmarks to a new file

I'm going to add another big benchmark and decided that 6000 lines was big
enough for this file.

Epic: none
Release note: none

stevendanna committed:
kvserver: add benchmark for IntentResolution

This benchmark creates intents for transactions in an assortment of states that
need to then be resolved by a reader.

Fixes #156796
Release note: none

stevendanna committed:
wip: comment to unconfuse us

stevendanna committed:
wip

stevendanna committed:
wip

stevendanna committed:
lock_table WIP

stevendanna committed:
lock_table: update test to match new resolve behaviour

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/159292)
<!-- Reviewable:end -->

stevendanna committed:
wip: no more fixed timestamp in benchmark

stevendanna committed:
wip: TestConcurrencyBasic works

miraradeva committed:
wip: lock table waiter tests pass, new claimantTxnFor

<what was there before: Previously, ...>
<why it needed to change: This was inadequate because ...>
<what you did about it: To address this, this patch ...>
